### PR TITLE
Group some print()s in backend_ps.

### PR DESCRIPTION
**Pull request by anntzer**
This results in some comments being reordered, but that's fine per
"PostScript Language Document Structuring Conventions Specification" v3
section 4.4,

    The order of some comments in the document is significant, but in a
    section of the document they may appear in any order. For example, in the
    header section, %%DocumentResources:, %%Title:, and %%Creator: may
    appear in any order.

## PR Checklist

- [ ] Has Pytest style unit tests
- [ ] Code is PEP 8 compliant
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

**Comment by phobson** [comment]
Should we be logging instead of printing?

**Comment by anntzer** [comment]
no, these go to the actual ps file :)

**Comment by phobson** [comment]
oh derp

File: lib/matplotlib/backends/backend_ps.py [hunk 1]
```diff
@@ -1080,10 +1080,12 @@ def print_figure_impl(fh):
             if isEPSF:
                 print("%!PS-Adobe-3.0 EPSF-3.0", file=fh)
             else:
-                print("%!PS-Adobe-3.0", file=fh)
+                print("%!PS-Adobe-3.0\n"
+                      "%%DocumentPaperSizes: {papertype}\n"
```
Comment (timhoffm) [inline lib/matplotlib/backends/backend_ps.py]: Needs a `.format(papertype)`

Comment (anntzer) [reply lib/matplotlib/backends/backend_ps.py]: indeed, fixed

File: lib/matplotlib/backends/backend_ps.py [hunk 2]
```diff
@@ -1092,14 +1094,14 @@ def print_figure_impl(fh):
...
-                print("%%Pages: 1", file=fh)
-            print("%%EndComments", file=fh)
+            print("%%Creator: {creator_str}\n"
+                  "%%CreationDate: {source_date}\n"
+                  "%%Orientation: {orientation}\n"
+                  "%%BoundingBox: {bbox[0]} {bbox[1]} {bbox[2]} {bbox[3]}\n"
+                  "%%EndComments\n"
+                  .format(creator_str=creator_str, source_date=source_date,
+                          orientation=orientation, bbox=bbox),
+                  end="", file=fh)
```
Comment (eric-wieser) [inline lib/matplotlib/backends/backend_ps.py]: Could just spell it `fh.write` at this point

Comment (anntzer) [reply lib/matplotlib/backends/backend_ps.py]: Then I'd rather change the whole thing to use fh.write throughout, but... too lazy.

**Comment by jklymak** [comment]
This is obviously failing all the tests.  Did you want to move forward w/ it or should we close it?

**Comment by anntzer** [comment]
should be fixed

**Pull request merged**